with recent attrs test_base_utils.test_attrs_with_repr fails

```
    def test_attrs_with_repr():
        @attrs
        class WithRepr(object):
            def __init__(self, x):
                self.x = x

            def __repr__(self):
                return 'foo'

>       assert hash(WithRepr(1)) == hash(WithRepr(1))
E       TypeError: unhashable type: 'WithRepr'
```

This is caused by https://github.com/python-attrs/attrs/issues/136; other tests work fine, so it seems `attrs` change doesn't affect eli5. But I can't say it for sure because I don't understand what this test was introduced for in a first place.

--- a/tests/test_base_utils.py
+++ b/tests/test_base_utils.py
@@ -38,6 +38,9 @@ def test_attrs_without_default():
 
 
 def test_attrs_with_repr():
+    """ Check that if __repr__ or __hash__ is defined, attrs does not override
+    them (there is a special logic for it in @attrs).
+    """
 
     @attrs
     class WithRepr(object):
@@ -47,11 +50,17 @@ def test_attrs_with_repr():
         def __repr__(self):
             return 'foo'
 
-    # assert hash(WithRepr(1)) == hash(WithRepr(1))
+        def __hash__(self):
+            return 1
+
+    assert WithRepr(1) == WithRepr(1) != WithRepr(2)
     assert repr(WithRepr(2)) == 'foo'
+    assert hash(WithRepr(2)) == 1
 
 
 def test_bad_init():
+    """ Constructor argument names must match attribute names.
+    """
 
     @attrs
     class BadInit(object):

